docs/administration-guide/aggregation/ingest-example.md

```diff
@@ -0,0 +1,607 @@
+# Ingest Aggregation Example
+
+## Simple Aggregation
+
+To demonstrate basic aggregation at ingest we can take the following graph as a
+start point and modify the schema so that the properties are summed together:
+
+```mermaid
+graph LR
+    A(["Person
+
+        ID: Dave"])
+    --
+    "Commit
+     added: 6
+     removed: 8"
+    -->
+    B(["Repository
+
+        ID: r1"])
+    A
+    --
+    "Commit
+     added: 35
+     removed: 10"
+    -->
+    B
+```
+
+As you can see we have two vertex types, `Person` and `Repository` both without
```

Comment:
These are actually two **Entity groups** called `Person` and `Repository`. They both use the similar vertex types, called `id.person.string` and `id.repo.string`, even though these could be the same type.